src/Speckle.Objects/Data/GisObject.cs

```diff
@@ -0,0 +1,14 @@
+using Speckle.Sdk.Models;
+
+namespace Speckle.Objects.Data;
+
+/// <summary>
```

Comment:
Comment should be updated - I will add this class in another pr, so no need to fix